src/applications/check-in/locales/en/translation.json

```diff
@@ -436,10 +436,10 @@
   "if-youre-claiming-mileage-file-online-now--UPDATE": "If you’re only claiming mileage, you can file online right now. We’ll just ask you a few questions—you won’t need receipts.",
   "if-claiming-other-expenses-file-online-or-mail-or-in-person--UPDATE": "If you’re claiming other expenses, like lodging, meals, or tolls, you will need receipts for these expenses. You can file online through the Beneficiary Travel Self Service System (BTSSS), by mail, or in person.",
   "penalty-statement--UPDATE": "<0>Penalty statement:</0> There are severe criminal and civil penalties, including a fine, imprisonment, or both, for knowingly submitting a false, fictitious, or fraudulent claim.",
-  "if-you-need-submit-receipts-other-expenses--UPDATE": "<0>If you need to submit receipts for other expenses like tolls, meals, or lodging, you can’t file a claim in this tool right now.</0> But you can file your claim online, within 30 days, through the Beneficiary Travel Self Service System (BTSSS). You can also file by mail or in person using VA Form 10-3542.",
-  "if-you-need-to-submit-receipts-for-other-expenses--helptext--UPDATE": "If you need to submit receipts for other expenses like tolls, meals, or lodging, you can’t file a claim now. But you can file your claim online, within 30 days, through the Beneficiary Travel Self Service System (BTSSS). You can also file by mail or in person using VA Form 10-3542.",
-  "if-you-traveled-by-bus-train-taxi-or-other-authorized--help-text--UPDATE": "<0>If you traveled by bus, train, taxi, or other authorized public transportation, you can’t file a claim in this tool right now.</0> But you can file your claim online, within 30 days, through the Beneficiary Travel Self Service System (BTSSS). You can also file by mail or in person using VA Form 10-3542.",
-  "if-you-traveled-from-a-different-address-you--helptext--UPDATE": "<0>If you traveled from a different address, you can’t file a claim in this tool right now.</0> But you can file your claim online, within 30 days, through the Beneficiary Travel Self Service System (BTSSS). You can also file by mail or in person using VA Form 10-3542.",
+  "if-you-need-submit-receipts-other-expenses--UPDATE": "<0>If you need to submit receipts for other expenses like tolls, meals, or lodging, you can’t file a claim in this tool right now.</0> But you can file your claim online through the Beneficiary Travel Self Service System (BTSSS). You can also file by mail or in person using VA Form 10-3542.",
```

Comment:
These keys will need to change in order to be picked up by Lori's script, so something like --update-30-days.